src/Lecture2.hs

```diff
@@ -181,7 +262,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (_:[]) = True
+isIncreasing (a:b:list) = a <= b && isIncreasing list
```

Comment:
I see that this function is incorrect 🤔 
It will return `True` on the list `[1, 2, 1]` but it should return `False`.

Reply:
Oh no! You're right. Thanks for catching the bug